models/DataObject/ClassDefinition/Data/ReverseObjectRelation.php

```diff
@@ -178,11 +179,6 @@ public function resolveDependencies(mixed $data): array
         return [];
     }
 
-    public function isOptimizedAdminLoading(): bool
-    {
-        return true;
-    }
```

Comment:
@BlackbitDevs I think we cannot remove this, because it's not internal

Reply:
Actually it is not removed but is now gets inherited from the parent class. If you want we can set this explicitly to `false` in `ReverseObjectRelation` but the whole purpose of this PR is to make reverse relations work the same as usual many-to-many object relations.

Reply:
@BlackbitDevs No it's fine. Thanks for the explanation